test/e2e/util.go

```diff
@@ -772,14 +772,14 @@ func runKubectl(args ...string) string {
 }
 
 // testContainerOutput runs testContainerOutputInNamespace with the default namespace.
-func testContainerOutput(scenarioName string, c *client.Client, pod *api.Pod, expectedOutput []string) {
-	testContainerOutputInNamespace(scenarioName, c, pod, expectedOutput, api.NamespaceDefault)
+func testContainerOutput(scenarioName string, c *client.Client, pod *api.Pod, containerId int, expectedOutput []string) {
+	testContainerOutputInNamespace(scenarioName, c, pod, containerId, expectedOutput, api.NamespaceDefault)
 }
 
 // testContainerOutputInNamespace runs the given pod in the given namespace and waits
 // for the first container in the podSpec to move into the 'Success' status.  It retrieves
 // the container log and searches for lines of expected output.
-func testContainerOutputInNamespace(scenarioName string, c *client.Client, pod *api.Pod, expectedOutput []string, ns string) {
+func testContainerOutputInNamespace(scenarioName string, c *client.Client, pod *api.Pod, containerId int, expectedOutput []string, ns string) {
```

Comment:
Because i want to bring in a test cases involved multiple containers, and @pmorie suggests me to commit this enhancement idea in a separate pr.